### PR TITLE
[Term Entry] Kotlin Math .sinh()

### LINKED ISSUE #3630: [Term Entry] Kotlin Math .sinh()
### Reason/inspiration (optional)

We would like a new term entry on the `.sinh()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/sinh/sinh.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by rahul0x00**
### Description

Added Term Entry for Kotlin Math.sinh()
Closes #3630 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/sinh/sinh.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Math.sinh()'
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sinh/sinh.md:2]:
```suggestion
Title: 'sinh()'
```

File: content/kotlin/concepts/math-methods/terms/sinh/sinh.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Description: 'Calculates the hyperbolic sine of a given number.'
+Subjects:
+  - 'Mathematics'
+  - 'Programming'
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sinh/sinh.md:6]:
```suggestion
  - 'Code Foundations'
```

Comment (caupolicandiaz) [reply content/kotlin/concepts/math-methods/terms/sinh/sinh.md:6]: There should only be items from our subjects.md listed here (2 item max)

File: content/kotlin/concepts/math-methods/terms/sinh/sinh.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`Math.sinh()`** method in Kotlin's `Math` class is used to calculate the hyperbolic sine of a given number.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sinh/sinh.md:16]:
```suggestion
The **`math.sinh()`** method in Kotlin's `math` class is used to calculate the hyperbolic sine of a given number.
```

File: content/kotlin/concepts/math-methods/terms/sinh/sinh.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Syntax
+
+```pseudo
+val hyperbolicSine = Math.sinh(number)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sinh/sinh.md:21]:
```suggestion
import kotlin.math.sinh
val hyperbolicSine = sinh(number)
```

File: content/kotlin/concepts/math-methods/terms/sinh/sinh.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+- `number`: The number for which you want to calculate the hyperbolic sine.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sinh/sinh.md:24]:
```suggestion
- `number`: The number, of type `Double` or `Float`, to be used in the calculation of the hyperbolic sine.
```

File: content/kotlin/concepts/math-methods/terms/sinh/sinh.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+The `Math.sinh()` method returns a value of type `Double` representing the hyperbolic sine of the input number.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sinh/sinh.md:26]:
```suggestion
```

File: content/kotlin/concepts/math-methods/terms/sinh/sinh.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+## Example
+
+This example shows how to use the `Math.sinh()` method to calculate the hyperbolic sine of a number:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sinh/sinh.md:30]:
```suggestion
This example shows how to use the `sinh()` method to calculate the hyperbolic sine of a number:
```

File: content/kotlin/concepts/math-methods/terms/sinh/sinh.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+```kotlin
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sinh/sinh.md:32]:
```suggestion
```kotlin
import kotlin.math.sinh
```

File: content/kotlin/concepts/math-methods/terms/sinh/sinh.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+fun main() {
+    val number = 2.0
+
+    val hyperbolicSine = Math.sinh(number)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sinh/sinh.md:36]:
```suggestion
    val hyperbolicSine = sinh(number)
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@rahul0x00 I've added my comments. Please update, and tag me for another review.

**Comment by rahul0x00** [comment]
@caupolicandiaz updated the entry

**Review by caupolicandiaz — APPROVED** [review]
@rahul0x00 This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@rahul0x00 I made one minor spacing edit. Now it looks good to go.

**Pull request merged**